src/js/utils/__tests__/ReducerUtil-test.js

```diff
@@ -0,0 +1,163 @@
+const ReducerUtil = require('../ReducerUtil');
+
+const idReducer = function (state = '', action) {
+  if (action.path.join('') === 'id') {
```

Comment:
Alright, that makes sense. No, let's keep the array. I was just curious to the format.
I think it makes sense, so you can use functions like `action.path.includes('container')`.